tools/buildsys/src/args.rs

```diff
@@ -29,6 +29,8 @@ const REBUILD_VARS: [(&str, u8); 15] = [
     ("BUILDSYS_TIMESTAMP", VARIANT),
     ("BUILDSYS_VARIANT", VARIANT),
     ("BUILDSYS_VERSION_BUILD", VARIANT),
+    ("BUILDSYS_VERSION_BUILD_TIMESTAMP", VARIANT),
+    ("BUILDSYS_VERSION_BUILD_EPOCH", VARIANT),
```

Comment:
We should re-drive all the package builds if Epoch changes, because newly built packages may end up with dependencies on previous package builds that assume the Epoch is set as expected:
```suggestion
    ("BUILDSYS_VERSION_BUILD_EPOCH", PACKAGE),
```